src/api/api/BIOMD/BIOMD.js

```diff
@@ -86,7 +86,6 @@ module.exports.CREATE_RECORD = async function (req, dbClient) {
 
   //Decleration
   var invalidCollection = false;
-  var id = new Object();
   let collection = req.Request.collection;
```

Comment:
@irushavm  Could this not be generalized so that any record can be created in a collection? I am thinking if we change the last parameter to collection. This would avoid changes for each new collection. Thoughts?

Reply:
While feeding collection into FIND_RECORDS is fine, I think controlling which collections to create should be something the server should do. The frequency of adding collections should be low. I put in a new change where CREATE_RECORD uses the `SUPPORTED_COLLECTIONS` array to determine if a collection is supported. This way, allowing a collection to be created is a one-line change.